src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -60,22 +61,39 @@ private class RenameRewriter : CSharpSyntaxRewriter
             private readonly CancellationToken _cancellationToken;
 
             private readonly ISymbol _renamedSymbol;
-            private readonly IAliasSymbol _aliasSymbol;
+            private readonly IAliasSymbol? _aliasSymbol;
             private readonly Location _renamableDeclarationLocation;
 
             private readonly RenamedSpansTracker _renameSpansTracker;
             private readonly bool _isVerbatim;
             private readonly bool _replacementTextValid;
-            private readonly bool _isRenamingInStrings;
-            private readonly bool _isRenamingInComments;
-            private readonly ISet<TextSpan> _stringAndCommentTextSpans;
             private readonly ISimplificationService _simplificationService;
             private readonly ISemanticFactsService _semanticFactsService;
             private readonly HashSet<SyntaxToken> _annotatedIdentifierTokens = new HashSet<SyntaxToken>();
             private readonly HashSet<InvocationExpressionSyntax> _invocationExpressionsNeedingConflictChecks = new HashSet<InvocationExpressionSyntax>();
 
             private readonly AnnotationTable<RenameAnnotation> _renameAnnotations;
 
+            /// <summary>
+            /// Flag indicating if we should perform a rename inside string literals.
+            /// </summary>
+            private readonly bool _isRenamingInStrings;
+
+            /// <summary>
+            /// Flag indicating if we should perform a rename inside comment trivia.
+            /// </summary>
+            private readonly bool _isRenamingInComments;
+
+            /// <summary>
+            /// A map from spans of tokens needing rename within strings or comments to an optional
+            /// set of specific sub-spans within the token span that
+            /// have <see cref="_originalText"/> matches and should be renamed.
+            /// If this sorted set is null, it indicates that sub-spans to rename within the token span
+            /// are not available, and a complete regex match should be performed to rename
```

Comment:
what is a 'complete regex match'?

Reply:
Meant a full regex match. I have removed 'complete' from the documentation.